tool/tsh/tsh.go

```diff
@@ -1582,24 +1582,12 @@ func onLogin(cf *CLIConf) error {
 		fmt.Fprintf(os.Stderr, "%s\n\n", warning)
 	}
 
-	// get any "on login" alerts
-	alerts, err := tc.GetClusterAlerts(cf.Context, types.GetClusterAlertsRequest{
-		Labels: map[string]string{
-			types.AlertOnLogin: "yes",
-		},
-	})
-	if err != nil && !trace.IsNotImplemented(err) {
-		return trace.Wrap(err)
+	if err := common.ShowClusterAlerts(cf.Context, tc, os.Stderr, map[string]string{
```

Comment:
Looks like we're only showing these on login. What about `tsh status`?

Reply:
I was under the impression from the discussion in the RFD PR that we were leaving out the tsh status alert warnings due to latency concerns

Reply:
Didn't we introduce the timeout specifically to alleviate these concerns?

Reply:
Ah you are right. Have added the expired license warnings to tsh status